readme.md

```diff
@@ -490,6 +490,7 @@
 - [CSS Learning](https://github.com/micromata/awesome-css-learning#readme) - Mainly about CSS – the language and the modules.
 - [Product Management](https://github.com/dend/awesome-product-management#readme) - Learn how to be a better product manager.
 - [Roadmaps](https://github.com/liuchong/awesome-roadmaps#readme) - Gives you a clear route to improve your knowledge and skills.
+- [YouTubers](https://github.com/JoseDeFreitas/awesome-youtubers#readme) - Watch videos and playlists from YouTubers that teach you about technology.
```

Comment:
I would use *Watch videos that teach you about technology*. Not sure though. Maybe adding YouTube somewhere.

Reply:
I agree. "Watch playlists" doesn't sound ok. I may change it to "_Watch videos from YouTubers that teach you about technology_". Though, as the list title is "YouTubers" I'd prefer keeping that word instead of "YouTube". If I would've to use "YouTube" then it should be something like "_Watch videos from YouTube channels that teach you about technology_", which in fact isn't that bad, but still prefer the first I wrote in this comment.

Apart from that, I've also thought about "_Watch video tutorials from YouTubers that teach you about technology_", as the content in the list are tutorials, practical and theoretical classes, guides, courses, etc.

What do you think?

Reply:
I've decided to set this:
> Watch video tutorials from YouTubers that teach you about technology.

I've removed the word "playlists" as it may sound strange. I also added the word "tutorials" to let know people you can learn from those videos. I prefer to keep the "YouTubers" word, as it is the same used in the list name and it's a recognized word.